src/components/member-entries/MemberEntries.tsx

```diff
@@ -61,11 +61,18 @@ export const MemberEntries = (props: MemberEntriesProps) => {
   ];
 
   const [memberSort, setMemberSort] = useState("firstName");
+  type MemberKeysValues =
```

Comment:
Since your variabele is called `memberSort` I would call your type something accordingly. MemberSortValues or something.